avocado/core/job.py

```diff
@@ -646,6 +647,17 @@ def run_tests(self):
 
         return self.exitcode
 
+    def _check_test_suite_uniqueness(self):
+        names = []
+        for suite in self.test_suites:
+            if suite.name not in names:
+                names.append(suite.name)
```

Comment:
I was wondering, does that make sense to loop over the whole list of suites and add all the duplicate names on a set? I hope this would be a rare case, but this way, Avocado would display all the duplicated names just once, avoiding the same error multiple times until the user fix all the test suite names.

Reply:
Good point, let's do that.